shared-lib/spiutils/src/protocol/firmware.rs

```diff
@@ -322,3 +437,104 @@ impl ToWire for WriteChunkResponse {
         Ok(())
     }
 }
+
+// ----------------------------------------------------------------------------
+
+wire_enum! {
+    /// When to perform the reboot.
+    pub enum RebootTime: u8 {
```

Comment:
Similarly to Johnathan's earlier comment, would this be better phrased as an Option?

Reply:
Actually, we don't need or want the `Unknown` value. I removed it. Good catch!